ctapipe/tools/train_energy_regressor.py

```diff
@@ -57,10 +57,17 @@ class TrainEnergyRegressor(Tool):
         default_value=0, help="Random seed for sampling and cross validation"
     ).tag(config=True)
 
+    n_jobs = Int(
+        default_value=None,
+        allow_none=True,
+        help="Number of threads to use for the reconstruction. This overwrites the values in the config",
```

Comment:
`In the config of each reconstructor`?

This value can also be set via the config